target/dovecot/10-ssl.conf

```diff
@@ -42,11 +42,15 @@ ssl_key = </etc/dovecot/ssl/dovecot.key
 # auth_ssl_username_from_cert=yes.
 #ssl_cert_username_field = commonName
 
-# DH parameters length to use.
-ssl_dh_parameters_length = 2048
-
-# SSL protocols to use
-ssl_protocols = !SSLv3,!TLSv1,!TLSv1.1
+# SSL DH parameters
+# Generate new params with `openssl dhparam -out /etc/dovecot/dh.pem 4096`
+# Or migrate from old ssl-parameters.dat file with the command dovecot
+# gives on startup when ssl_dh is unset.
+ssl_dh = </etc/dovecot/dh.pem
+
+# Minimum SSL protocol version to use. Potentially recognized values are SSLv3,
+# TLSv1, TLSv1.1, and TLSv1.2, depending on the OpenSSL version used.
+ssl_min_protocol = TLSv1.2
```

Comment:
Why did you bump this to 1.2?

This way it prevents newer versions like 1.3. Please leave the old notation.

Reply:
As I understand it, the old config said: Any version that is not SSLv3 or TLSv1 or TLSv1.1 (which should mean TLSv1.2 or newer)
The new config says: minimum version is TLSv1.2 (it does not prevent newer versions and should be the same as before).

The old "ssl_protocols" is not supported anymore. We have to use the new "ssl_min_protocol"! (see https://wiki2.dovecot.org/Upgrading/2.3):

> ssl_protocols setting was replaced by ssl_min_protocol. Now you only specify the minimum ssl protocol version Dovecot accepts, defaulting to TLSv1.